### PR TITLE
Use postcss-preset-env instead of cssnext #239

### PR DESCRIPTION
**Pull request by ur300**
postcss-preset-env added to the main page instead of cssnext

### CODE REVIEW

**Comment by ai** [comment]
@ur300 it is always better to mention main maintainer in New pull request

/cc @marcustisater

**Comment by marcustisater** [comment]
Thanks for PR @ur300 👍Remember to reference the issue next time. Let me look through code and copy.

**Comment by marcustisater** [comment]
Whops sorry my fault @ur300. You did reference it in the title 😄

Code looks good but I think the SVG should have a set height and width with correct viewbox like https://github.com/postcss/postcss.org/blob/master/web_modules/InANutshell/css-modules.svg has. Would you mind fixing that? That would be awesome! 

@ai @jonathantneal is the copy ok? See attacthed screenshot. 
order 

On the other hand I think the order should be like before, so

1. Autoprefixer.
2. Preset Env 
3. CSS Modules
4. Stylelint
5. LostGrid

File: web_modules/InANutshell/index.css [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 .logo {
   margin-right: 1rem;
+  max-width: 80px;
```
Comment (marcustisater) [inline web_modules/InANutshell/index.css]: Remove this. Set width and height through SVG.

**Comment by ai** [comment]
Yeap. Let's more Preset Env to second position.

And let's remove `pv-` from media query name to make it simpler.

File: web_modules/InANutshell/index.css [hunk 2]
```diff
@@ -127,11 +128,12 @@
 
   .infoAlt {
     composes: info;
-    margin-left: 1.5rem;
+    margin-left: 0.5rem;
```
Comment (marcustisater) [inline web_modules/InANutshell/index.css]: I think this looks a bit too tight.

**Comment by jonathantneal** [comment]
This all seems good to me.

**Comment by ur300** [comment]
hi @marcustisater, thanks for review. all the fixes are done - svg size, margin, order of features, media query name.

**Comment by ai** [comment]
@ur300 task is marked as solved by you https://cultofmartians.com/tasks/postcss-preset-env-postcss.html

**Comment by jonathantneal** [comment]
@ai, sweet, is this being merged today?

**Comment by ai** [comment]
We waiting for @marcustisater review and deploy

**Comment by marcustisater** [comment]
SVG viewbox is still odd, but I can fix it later. LGTM

**Comment by marcustisater** [comment]
@ai @jonathantneal bah, something is wrong with deploy, I think it has something to do with the node env or when we switched over to HTTPS. Let me check tonight. Sorry for troubles! I will let you know when it's fixed.

**Comment by marcustisater** [comment]
@ai @jonathantneal I deployed from local env, should be up live now 😄  I will add new issue for auto deploy trouble. 😓

**Pull request merged**